### PR TITLE
chore: update notify msg

### PR DESCRIPTION
**Pull request by catngh**
**What does this PR do?**

- Add server link to new guild notification
- Update new upvote msg

Upvote message for unknown users
image
Link to new server
image

### CODE REVIEW

File: pkg/service/discord/discord.go [hunk 1]
```diff
@@ -52,6 +52,20 @@ func (d *Discord) NotifyNewGuild(guildID string, count int) error {
 	if err != nil {
 		return fmt.Errorf("failed to get guild info: %w", err)
 	}
+	inviteUrl := ""
+	channels, err := d.session.GuildChannels(guildID)
+	if err != nil {
+		return fmt.Errorf("failed to get guild channels: %w", err)
```
Comment (trkhoi) [inline pkg/service/discord/discord.go:58]: Should log error

**Review by trkhoi — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]

File: pkg/entities/webhook.go [hunk 2]
```diff
@@ -453,7 +453,12 @@ func (e *Entity) WebhookUpvoteStreak(userID, source string) error {
 
 	e.handleUpvoteXPBonus(streak)
 	e.logUserUpvote(userID, source, chatDate)
-	e.svc.Discord.SendUpvoteMessage(userID, source)
+	isStranger := false
+	_, err = e.repo.Users.GetOne(userID)
+	if err != nil {
```
Comment (anhnh12) [inline pkg/entities/webhook.go:458]: isStranger := err != nil

File: pkg/service/discord/discord.go [hunk 3]
```diff
@@ -50,8 +50,24 @@ func (d *Discord) NotifyNewGuild(guildID string, count int) error {
 	// get new guild info
 	guild, err := d.session.Guild(guildID)
 	if err != nil {
+		d.log.Errorf(err, "[discord.NotifyNewGuild] - failed to get guild: %s", guildID)
 		return fmt.Errorf("failed to get guild info: %w", err)
 	}
+	inviteUrl := ""
+	channels, err := d.session.GuildChannels(guildID)
```
Comment (anhnh12) [inline pkg/service/discord/discord.go:57]: should create `func generateGuildInviteLink()`

**Review by anhnh12 — CHANGES_REQUESTED** [review]

File: pkg/entities/webhook.go [hunk 4]
```diff
@@ -453,7 +453,9 @@ func (e *Entity) WebhookUpvoteStreak(userID, source string) error {
 
 	e.handleUpvoteXPBonus(streak)
 	e.logUserUpvote(userID, source, chatDate)
-	e.svc.Discord.SendUpvoteMessage(userID, source)
+	_, err = e.repo.Users.GetOne(userID)
```
Comment (anhnh12) [inline pkg/entities/webhook.go:456]: I think we should add a case here
```
if err != nil && err != gorm.ErrRecordNotFound {
log.........
return err
}
isStranger := err != nil
```

**Review by anhnh12 — APPROVED** [review]

**Pull request merged**